File: Week-1/LonelyInteger/src/main/java/MyCode.java
import java.util.Collections;
import java.util.List;
import java.util.stream.Collectors;

public class MyCode {
    public static int lonelyinteger(List<Integer> a) {
        int lonely = a.stream().filter(i -> a.stream().filter(it -> it == i).count() == 1).collect(Collectors.toList()).get(0);
        return lonely;
    }
}

File: Week-1/LonelyInteger/src/main/kotlin/MyCode.kt

fun lonelyInteger(a: Array<Int>): Int{
    var lonely = 0
    for(i in a){
        if(a.count{it == i} == 1){
            lonely = i
        }
    }
    return lonely
}
fun main(args: Array<String>) {
    val n = readLine()!!.trim().toInt()

    val a = readLine()!!.trimEnd().split(" ").map{ it.toInt() }.toTypedArray()

    val result = lonelyInteger(a)

    println(result)
}